internal/configs/version1/nginx.tmpl

```diff
@@ -34,7 +34,11 @@ http {
     {{- end}}
 
     {{if .LogFormat -}}
-    log_format  main  '{{.LogFormat}}';
+    log_format  main {{if .LogFormat -}}
```

Comment:
the `{{if .LogFormat -}} ... {{end}}` is not required here - using just range is enough. 
the same applies for the stream log format.